### PR TITLE
Remove faulty data placed in credential data.

### PR DESCRIPTION
**Pull request by gannan08**
* Localize credential image size logic.
* Require use of an Issuer Map that maps IDs to an object
  containing more refined data.
* Update issuer-logo.png image to a higher res image.
* Only display issuer's name if it exists when using the
  `CredentialDetails` component.
* Update priority order of which image to display on the
  credential. Order: `credential.image` > `issuer.image` > `issuer.logo`.

### CODE REVIEW

File: credentialMixin.js [hunk 1]
```diff
@@ -4,33 +4,15 @@
...
-      _createFields(fields, credentialSubject, this.schema);
-      return fields;
-    }
-  },
   props: {
     credential: {
       type: Object,
       required: true,
     },
+    didMap: {
```
Comment (dlongley) [inline credentialMixin.js]: DIDs shouldn't be required, a non-DID identifier should work. Let's call this `issuerMap` instead.

**Review by dlongley — CHANGES_REQUESTED** [review]

**Comment by gannan08** [comment]
Addresses https://github.com/digitalbazaar/bedrock-vue-credential-card/issues/11

**Review by dlongley — APPROVED** [review]
LGTM modulo the changes I mentioned out-of-band.

**Comment by gannan08** [comment]
> LGTM modulo the changes I mentioned out-of-band.

Issues have been addressed. Will update changelog and proceed with a major release.

**Pull request merged**